### PR TITLE
Add HW3 Test - Init Check Edge Case

### PR DESCRIPTION
**Pull request by ido18501**
Add HW3 2b Test - same distance on initial check

### CODE REVIEW

File: Tests/Homework3/2b/student_tests.json [hunk 1]
```diff
@@ -6,16 +6,7 @@
       "output": "Please enter array length:\nPlease enter target number:\nPlease enter k:\nPlease enter sorted and uniq array:\n1"
     },
     {
-      "name": "min int test",
```
Comment (PNone) [inline Tests/Homework3/2b/student_tests.json]: Replaced with different test, thx though

File: Tests/Homework3/2b/student_tests.json [hunk 2]
```diff
@@ -6,16 +6,7 @@
...
-      "input": "3 -2147483648 2 -2147483648 0 2000000000",
```
Comment (PNone) [inline Tests/Homework3/2b/student_tests.json]: Undo this change please

**Review by PNone — CHANGES_REQUESTED** [review]
